src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,41 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+ 
+	private static final int FULL_STACK = 11;

```

Comment:
This constant is related to NUMBERS_MAX. Changing only one leads to an error (inconsistency).